shared/roughtime/roughtime.go

```diff
@@ -52,12 +52,15 @@ func recalibrateRoughtime() {
 
 	// Log Debug Results.
 	for _, res := range results {
+		if res.Error() != nil {
+			log.Errorf("Could not get rough time result: %v", res.Error())
+			break
```

Comment:
```suggestion
			continue
```
Actually you had it right, sorry!